.gitignore

```diff
@@ -3,4 +3,8 @@ path/to/venv/
 */__pycache__/
 */*/__pycache__/
 .env
-/venv/
\ No newline at end of file
+/venv/
+
+# main_repo.db and repo_fam.db are created by database.py to store the state of the repository
+main_repo.db
+repo_fam.db
```

Comment:
You ignore files here, but this PR now adds these files:

```
db-opentensor/main_repo.db
db-opentensor/repo_fam.db
```
Which I don't think should be the case.

Reply:
removed, i added them by mistake